allure-generator/src/main/javascript/components/graph-pie-chart/PieChartView.js

```diff
@@ -34,6 +34,11 @@ class PieChartView extends BaseChartView {
       value: stats[key],
       part: stats[key] / total,
     }));
+    for (let i = 0; i < this.data.length; i++) {
```

Comment:
why not to use `.filter(item => item.value)` instead?

Reply:
ok i will do that !